### ISSUE TITLE
Julia registry

### ISSUE THREAD

**Issue by oameye** [JuliaDynamics/CriticalTransitions.jl#10]
Are you guys planning to add CriticalTransitions.jl to the general Julia Registry? That way the package can be installed through `Pkg.add("CriticalTransitions")`.

**Comment by reykboerner** [comment]
Yes, we are planning to but felt the package is currently still very much work in progress. Not sure when it's best practice to add a package to the registry but we still have a few things on our to-do list which we've just re-started working on.

**Comment by oameye** [comment]
I don't think that their any rules around that. Maybe even to it earlier than later, as you can already hog the name. We are planning to write a bridge to [HarmonicBalance](https://github.com/NonlinearOscillations/HarmonicBalance.jl) to give it stochastic functionalities. We are especially interested in the implementations of the gMAM.

**Comment by oameye** [comment]
I am a big fan of the package. If you guys would share you `to-do` list, I am happy to tackle some of them.

**Comment by reykboerner** [comment]
Cool, thanks a lot for your feedback and help! We'll post a to-do list here in the next days.

**Comment by oameye** [comment]
Bunp @reykboerner

**Comment by oameye** [comment]
Any chance we can release CriticalTransitions.jl to the Julia registry in the coming month>

**Comment by reykboerner** [comment]
As discussed @oameye @ryandeeley @raphael-roemer, we want to register the package with the Julia registry once we have working tests and complete docs for the currently exported functions.

**Comment by reykboerner** [comment]
@JuliaRegistrator register()

**Comment by reykboerner** [comment]
@oameye I sent this to register since I think the package is in a decent state now - we should hear back soon :) Unless you think we should cancel and wait till v1.0

**Comment by oameye** [comment]
No I think it is completely fine. Awesome :)

**Comment by reykboerner** [comment]
We're live on the registry 🥳

**Comment by oameye** [comment]
That's a milestone!!
